Examples/test-suite/cpp11_attribute_specifiers.i

```diff
@@ -22,9 +27,13 @@
 #pragma warning(disable : 5030) // attribute is not recognized ('likely' and 'unlikely')
 #endif
 
+#ifndef __BIGGEST_ALIGNMENT__
+#define __BIGGEST_ALIGNMENT__ 16
+#endif
```

Comment:
Defining a macro with a reserved name makes the program ill-formed so it'd be better to use e.g. `BIGGEST_ALIGNMENT`.  If it actually matters to use `__BIGGEST_ALIGNMENT__` when it is defined by the compiler we could define it using

```
#ifdef __BIGGEST_ALIGNMENT__
#define BIGGEST_ALIGNMENT __BIGGEST_ALIGNMENT__
#else
#define BIGGEST_ALIGNMENT 16
#endif
```

and then use `BIGGEST_ALIGNMENT` below.

Reply:
I don't think this is the only anti-pattern in the unit tests, this will only add additional clutter.

Reply:
Please change this.